Parameters:
 head_angle_referance: 0.001
 angular_speed_referance: 0.35
